.changeset/cold-lizards-reflect.md

```diff
@@ -0,0 +1,6 @@
+---
+'@udecode/plate-table': major
+'@udecode/plate-ui-table': major
+---
```

Comment:
We only use `major` for breaking changes. It's `minor` for new features (and very small breaking changes that are unlikely to affect anyone–use your discretion), and `patch` for bug fixes. 🙂